src/pages/index.js

```diff
@@ -172,3 +284,13 @@ addNewCardButton.addEventListener("click", () => {
   addCardPopup.open();
 });
 cardCloseButton.addEventListener("click", () => addCardPopup.close());
+deletePopupCloseButton.addEventListener("click", () => {
+  deleteCardPopup.close();
+});
+avatarPopupCloseButton.addEventListener("click", () => {
+  editAvatarPopup.close();
```

Comment:
Please, delete them

The close icon is handled only in `Popup`

Reply:
* You haven't deleted something https://github.com/Gerson773/se_project_aroundtheus/pull/1#discussion_r1361339510